### PR TITLE
Update Integration tests README

### PR DESCRIPTION
**Pull request by Rebits**
|Related issue|
|-------------|
|    #4711          |

## Description

This PR update the Integration Tests README file.

### Updated

- Update Integration tests README.

### CODE REVIEW

File: tests/integration/README.md [hunk 1]
```diff
@@ -32,41 +34,40 @@ systemctl stop firewalld
...
 
-# Install Python3
+```sheel script
 yum install python36 python36-pip python36-devel -y
-
-# Install Python libraries
-pip3 install pytest freezegun jq jsonschema pyyaml==5.3 psutil distro pandas==0.25.3 pytest-html==2.0.1 numpydoc==0.9.2
 ```
 
-- Add some internal options and restart
```
Comment (pro-akim) [inline tests/integration/README.md:51]: Apart from installing dependencies,
Shouldn't the setting of internal options be retained?

File: tests/integration/README.md [hunk 2]
```diff
@@ -79,68 +80,52 @@ echo 'monitord.rotate_log=0' >> $wazuh_path/etc/local_internal_options.conf
...
-- Install Python dependencies
+- Install integration tests dependencies
 
 ```shell script
-pip install pytest freezegun jsonschema pyyaml==5.4 psutil paramiko distro pywin32 pypiwin32 wmi pandas==0.25.3 pytest-html==2.0.1 numpydoc==0.9.2
+cd wazuh-qa
+python -m pip install -r requirements.txt
 ```
 
-- Change `time-reconnect` from `C:\Program Files (x86)\ossec-agent\ossec.conf`
```
Comment (pro-akim) [inline tests/integration/README.md:88]: More internal options to take into consideration.

File: tests/integration/README.md [hunk 3]
```diff
@@ -79,68 +80,52 @@ echo 'monitord.rotate_log=0' >> $wazuh_path/etc/local_internal_options.conf
...
-# Install dependencies
-brew install autoconf automake libtool
-
-# Install Python libraries
-pip3 install filetype freezegun jq jsonschema lockfile numpydoc psutil pytest-html pytest-testinfra pyyaml
+```shell script
+brew install python3
 ```
 
-- Add some internal options and restart
```
Comment (pro-akim) [inline tests/integration/README.md:123]: More internal options to take into consideration.

**Review by pro-akim — CHANGES_REQUESTED** [review]
Shouldn't the setting of internal options be retained?
Checking the pipeline currently running the process, these internal options remain effective when running the test.

**Comment by Rebits** [comment]
Regarding the local internal options. In order to streamline the maintenance process for local internal options, it is essential to ensure that integration tests possess the capability to configure every aspect necessary for their execution independently.

If a particular set of local configuration options or specific agent configurations is essential for all integration tests, it is advised to incorporate these configurations directly into the respective tests. This approach ensures clarity and self-sufficiency in each test.

However, it's acknowledged that there may be instances where certain options are required for outdated tests. For these reasons I will reverse those removal

File: tests/integration/README.md (hunk 3 above)
Comment (Rebits) [reply tests/integration/README.md:123]: Done 15215b391ba5d18eae8923b418137fd9f649b541

File: tests/integration/README.md (hunk 2 above)
Comment (Rebits) [reply tests/integration/README.md:88]: Done 15215b391ba5d18eae8923b418137fd9f649b541

File: tests/integration/README.md (hunk 1 above)
Comment (Rebits) [reply tests/integration/README.md:51]: Done 15215b391ba5d18eae8923b418137fd9f649b541

File: tests/integration/README.md [hunk 4]
```diff
@@ -32,101 +34,116 @@ systemctl stop firewalld
 systemctl disable firewalld
 ```
 
-- Install Python and its dependencies
+- Install Python3
+
+```sheel script
```
Comment (pro-akim) [inline tests/integration/README.md]: fix to shell

**Review by pro-akim — APPROVED** [review]
## Review Notes
LGTM

**Pull request merged**